drivers/issi/is31fl3741.h

```diff
@@ -23,14 +23,20 @@
 #include <stdbool.h>
 
 typedef struct is31_led {
-    uint8_t  driver : 2;
+#if __AVR__
```

Comment:
AVR can do bitfields too...

Reply:
I know. But it will generate really inefficient code, which I don't think justifiable.

Reply:
AVR is already pretty inefficient, this won't make much difference.

Reply:
good point. if you say so, let's remove it and make it simple.